common/utils/works.js

```diff
@@ -23,6 +25,32 @@ export function getProductionDates(work: Work) {
     .reduce((a, b) => a.concat(b), []);
 }
 
+export function getDownloadOptionsFromManifest(
+  iiifManifest: IIIFManifest
+): IIIFRendering[] {
+  const sequence = iiifManifest.sequences.find(
+    sequence => sequence['@type'] === 'sc:Sequence'
+  );
+  return (sequence && sequence.rendering) || [];
+}
+
+export function getDownloadOptionsFromImageUrl(
```

Comment:
Out of interesting, why is the function called download options, when we have explicitly called the `Rendering`s in it's type?

Reply:
'Rendering' comes from the iiifManifest. I made the image download options the same shape, but DownloadOptions seems more explanatory. Would you prefer they were all the same, and if so, which one?

Reply:
But if I wanted to use a `Rendering` elsewhere, I would probably write another function.
I'd say say what it does, and then you can chose what you want to do with it at render time.

Bikeshedding though.